portfolio/src/main/webapp/script.js

```diff
@@ -64,4 +64,49 @@ function scrollToSection(sectionId) {
         behavior: 'smooth'
     });
 	return false;
+}
+
+/**
+ * Lightbox Gallery Implementation
+ */
+
+// keeps track of current picture currently opened
+var currPic;
+
+// Displays Lightbox open user click
+function openLightbox(picIndex) {
+    document.getElementById("lightbox").style.display = "flex";
+    showPic(picIndex);
+}
+
+// Helper function to show specific picture in lightbox
+function showPic(picIndex) {
+  const picsArray = document.getElementsByClassName("lightbox-pic");
+
+  if (picIndex > picsArray.length) {
+    picIndex = 1;	
+  }
+  
+  else if (picIndex < 1) {
+  	picIndex = picsArray.length;
+  }
```

Comment:
Consider using the remainder operator (`%`) to keep your index within bounds here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators#Remainder

Reply:
This is a really clever way of wrapping around array index, thanks!!